code/game/machinery/cryopod.dm

```diff
@@ -340,7 +336,9 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 				else
 					mob_occupant.transferItemToLoc(items, loc, TRUE)
 
-	QDEL_LIST(mob_occupant.GetAllContents())
+	var/list/contents = list()
+	contents = mob_occupant.GetAllContents()
+	QDEL_LIST(contents)
```

Comment:
...What?

Reply:
What was wrong with the old thing? And if it was wrong, why are you making a new variable, and defining it again later?

Reply:
![Imgur img](https://i.imgur.com/NfH1hf2.png)
this was by your suggestion which you did not expand upon when asked about it
it returns the error ```proc call requires static type: "Cut"dreamchecker(proc_call_static_type)```

Reply:
```dm
var/list/contents = mob_occupant.GetAllContents()
QDEL_LIST(contents)
```